tests/helper/helper_cli.go

```diff
@@ -13,3 +13,16 @@ type CliRunner interface {
 	WaitAndCheckForExistence(resourceType, namespace string, timeoutMinutes int) bool
 	GetServices(namespace string) string
 }
+
+// NewCliRunner initializes new NewCliRunner with the appropriate platform
+func NewCliRunner(cli string) CliRunner {
```

Comment:
I don't think this is require because for kubernetes we do have `KUBERNETES` env var set. Also `OcRunner`  and `KubectRunner` can be used accordingly. For ex - 

```
var cliRunner helper.CliRunner

	// Using program commmand according to cliRunner in devfile
	if os.Getenv("KUBERNETES") == "true" {
		cliRunner = helper.NewKubectlRunner("kubectl")
	} else {
		cliRunner = helper.NewOcRunner("oc")
	}
```

So, I am not sure why we need this.

Reply:
I think its cleaner to have `cliRunner := NewCliRunner("kube")` & `cliRunner := NewCliRunner("docker")`

and in `NewCliRunner` func, do the checks you mentioned for either `oc` and `kubectl`; instead of having a if-else every time in the test we want to use `kubectl`.

But I'll make your suggested changes!